apps/cms/lib/page/agenda.ex

```diff
@@ -0,0 +1,44 @@
+defmodule CMS.Page.Agenda do
+  @moduledoc """
+  Represents an agenda content type in the Drupal CMS.
+  """
+
+  alias CMS.Field.Link
+  alias CMS.Partial.Paragraph
+
+  import CMS.Helpers,
+    only: [
+      field_value: 2,
+      int_or_string_to_int: 1,
+      parse_paragraphs: 3,
+      parse_link: 2
+    ]
+
+  defstruct id: nil,
+            title: "",
+            topics: [],
+            collect_info: false,
+            event_reference: nil,
+            formstack_url: nil
+
+  @type t :: %__MODULE__{
+          id: integer | nil,
+          title: String.t(),
+          topics: [Paragraph.AgendaTopic.t()],
+          collect_info: boolean,
+          event_reference: integer | nil,
```

Comment:
Technically the same, but I think specifying `CMS.Page.Event.id_t()` in place of `integer` would be more explicit.

Reply:
@thecristen sorry, what's `id_t()` (not currently in front of the code base so sorry it's it's just above this snippet, haha).

@hannahpurcell yes, in Drupal you can have an agenda node like you can have a basic page node, but they are never meant to be directly accessed. For an event, the agenda's "event_reference" field will be `nil` if no one has yet "linked" or "referenced" an event the agenda should be "attached" to -- it's not a hard requirement. For instance, they could work on an agenda async of the actual event.

Reply:
@amaisano 🤡  Upon further inspection please disregard my comment -- we use `.id_t()` for a few structs (Route, Shape, Trip...) and I thought it was some neat built-in feature but it's just that id_t is defined on those structs -- whoops